README.md

```diff
@@ -791,7 +791,7 @@ to any of your questions we recommend you
 
 # License
 
-Copyright (C) 2017-2022 [The ORT Project Authors](./NOTICE).
+Copyright (C) 2017-2023 [The ORT Project Authors](./NOTICE).
```

Comment:
Also see my comment at https://github.com/oss-review-toolkit/ort-config/pull/93#discussion_r1062310339. So should we simply drop the end year instead?

Reply:
I would be fine with keeping the end year in the README, not for legal reasons but to emphasize that the project is active.